_episodes/04-loop.md

```diff
@@ -147,7 +147,7 @@ for variable in collection:
 
 Using the oxygen example above, the loop might look like this:
 
-![loop_image](../fig/loops_image.png)
+![Graph illustrating a variable 'char' being assigned the value of each character in the word 'oxygen' in turn and then being printed](../fig/loops_image.png)
 
```

Comment:
Based on [WebAIM's tip](https://webaim.org/techniques/alttext/) not to use "image of ..." or "graphic of ..." what do you say to replacing "Graph illustrating a" with "Loop" to read: "Loop variable 'char' being assigned the value of each character in the word 'oxygen' in turn and then being printed"?

Reply:
Hi - yes, that's good.